.cirrus.yml

```diff
@@ -320,11 +320,18 @@ task:
       package_install_script:
         - apt-get update
         - apt-get install -y curl chromium
+        - apt install -y dbus
         - ln -s -T /usr/bin/chromium /usr/bin/chromium-browser
+        - mkdir -p /var/run/dbus
+        - dbus-daemon --config-file=/usr/share/dbus-1/system.conf --print-address
       env:
         CI_DISTRO: debian
         CI_MAIN_MODULE: /usr/lib/x86_64-linux-gnu/nss/libnssckbi.so
         CI_BAK_MODULE: /usr/lib/x86_64-linux-gnu/nss/libnssckbi.orig.so
+      matrix:
```

Comment:
I think you want this `matrix` to contain a list of objects, not just a single object.  The list should contain an object covering the  `env` for CKBI (moved from above) and another object covering the `env` for Softoken.